src/vault.py

```diff
@@ -438,6 +438,25 @@ def ddos(args):
 
     args = parser.parse_args()
 
+    if args.all:
+        if args.url:
+            url = check_url(args.url)
```

Comment:
What's the need to store URL in another variable? Instead `args.url = check_url(args.url)`

Reply:
@mzfr Actually we won't be using `url` variable anywhere, what gets passed to other modules is `args.url` and `args.ip`. So it was not needed! :)